sdks/python/apache_beam/runners/common.py

```diff
@@ -431,7 +530,13 @@ def _reraise_augmented(self, exn):
     raise new_exn, None, original_traceback
 
 
-class _OutputProcessor(object):
+class OutputProcessor(object):
+
+  def process_outputs(self, windowed_input_element, results):
+    raise NotImplementedError
```

Comment:
<!--new_thread; commit:7ca389e990c4482eec47aecdc1913eeee0b169f9; resolved:0-->
What is the motivation for this change?